modules/drivers/mongo/src/metabase/driver/mongo.clj

```diff
@@ -435,8 +436,8 @@
                               :expressions/datetime            true
                               :expressions/today               true
                               ;; Index sync is turned off across the application as it is not used ATM.
-                              :index-info                      false
-                              :database-routing                true}]
+                              :index-info false
+                              :database-routing true}]
```

Comment:
nit: alignment

Reply:
Out, I'm going to leave this one and ask for forgiveness. I want to avoid any more test reruns than are necessary due to time pressure.

IMO We can do a formatting pass later if it bothers somebody enough. This is certainly not the only example of misalignment I've seen in the code I've been changing so its like, the window is already broken :D